### ISSUE TITLE
Removal off Bootstrap dependency.

### ISSUE THREAD

**Issue by emil-palm** [otm/networkmap.js#14]
I was looking into integrating networkmap.js into our project which used bootstrap already and that will clash pretty damn hard.

as far as i can see its only used for 1 macro; "white" which would be simple enough to fix ourself.
And for the rest we should perhaps atleast build specific networkmap css styles so we dont clash with bootstrap.

Any thoughts?

**Comment by otm** [comment]
The networkmap.js css is basicly bootstrap (but a bit outdated version). Basicly I don't have any objections on cleaning that up. Do you have any more information about the changes you would like to do?

**Comment by emil-palm** [comment]
Yeah and only thing i can see we use otherwise then "white" is prettymuch the button styles so i was thinking of just removing all of bootstrap see whats broken and then do some copy/paste with renames to make the settings box look okay again.

**Comment by otm** [comment]
Do the change and lets see how it works.

**Comment by mrevilme** [comment]
Yeah i might do that the upcomming weeks now im working on building support for saving line states after changes.

**Comment by otm** [comment]
What do you mean wit "saving line states after changes"? If you mean saving
changes after the graph has been edited (moved nodes etc) there is already
support for that "onChange" configuration block in the configuration file.

Or is it something else?

//Nils
